WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -31,7 +31,7 @@ struct ItemCardView: View {
             VStack {
                 Text(item.name)
                     .foregroundStyle(Color.primaryBackground)
-                Text(item.price)
+                Text(item.formattedPrice)
```

Comment:
The price is already formatted from when we map from `Product` to `POSProduct` here: https://github.com/woocommerce/woocommerce-ios/blob/trunk/Yosemite/Yosemite/PointOfSale/POSProductProvider.swift#L58 , so this property wouldn't be needed.

Reply:
We need to separate clean price and formatted price because clean price is expected when creating/syncing the order to get a proper totals calculation

Reply:
Gotcha, in this case maybe makes sense to create the `POSProduct` with both? Rather than doing `price: formattedPrice` one property could just hold the _clean_ price, and the other the _formattedPrice_ when are created/mapped.